web/src/debug.ts

```diff
@@ -10,17 +10,23 @@
     evaluates to foo() and prints the elapsed time
     to the console along with the name "foo". */
 
-export function print_elapsed_time(name, fun) {
+export function print_elapsed_time(name: string, fun: () => number): number {
```

Comment:
This should use use a generic `T`, not `number`.

Reply:
Fixed it but there is a node test failing so im working on it